<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenesysWindow</class>
 <widget class="QMainWindow" name="GenesysWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Genesys LV Control</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Device setting</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Serial  Port</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sbDEVICE"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Address</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sbADDRESS"/>
       </item>
       <item>
        <widget class="QPushButton" name="pbDEVICE">
         <property name="text">
          <string>Set</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Commands</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pbOPEN">
         <property name="text">
          <string>OPEN</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbCLOSE">
         <property name="text">
          <string>CLOSE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbON">
         <property name="text">
          <string>ON</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbOFF">
         <property name="text">
          <string>OFF</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbSTATUS">
         <property name="text">
          <string>STATUS</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="tbSTATUS"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
